ds701_book/11-Dimensionality-Reduction-SVD-II.qmd

```diff
@@ -1,857 +1,759 @@
+We have seen some of these challenges already, in particular:
 
-Notice that $U$ contains a row for each data object.   
-
-::: {.content-visible when-profile="slides"}
-## Dimensionality reduction
-:::
-
-We can select the $k$ largest singular values and form the matrix:
+:::: {.incremental}
+- the curse of dimensionality, where data points become sparse in higher dimensions and distance metrics have less meaning,
+- overfitting, where high-dimensional data can lead models becoming overly complex and fitting to noise in the data as opposed to the actual signal,
+- computational complexity, high-dimensional data requires more computing power and memory,
+- visualization, where high-dimensional data makes understanding and interpreting the data difficult our data.
+::::
+--- 
 
-$$
-A^{(k)} =
-\begin{bmatrix}
-\vdots &  \vdots  \\
-\mathbf{u}_1 & \mathbf{u}_{k} \\
-\vdots& \vdots  \\
-\end{bmatrix}
-\begin{bmatrix}
-\sigma_{1} & &    \\
- & \ddots &    \\
-&  & \sigma_{k}  \\
-\end{bmatrix}
-\begin{bmatrix}
-\cdots & \mathbf{v}_1 & \cdots   \\
-\cdots & \mathbf{v}_k & \cdots   \\
-\end{bmatrix}.
-$$
+How can we reduce the dimension of our data but still preserve the most important information in our dataset?
 
 :::: {.fragment}
-By forming $A^{(k)}$ we have transformed the data objects that live in an $n$ dimensional space to a $k$ dimensional space, where $k$ is (probably much) smaller than $n$.
-::::
+We consider two techniques:
 
-:::: {.fragment}
-This is an example of __dimensionality reduction.__
+:::: {.incremental}
+- Principle Component Analysis (PCA)
+- t-distributed stochastic neighbor embedding (t-SNE)
 ::::
-
-:::: {.fragment}
-When we take our data to be the rows of $U\Sigma$ instead of the rows of $A$, we are __reducing the dimension__ of our data from $n$ dimensions to $k$ dimensions.
 ::::
 
 :::: {.fragment}
-We will see today that the SVD is the __optimal__ transformation of the data into $k$ dimensions. It is optimal in the sense that it captures the maximum variance in the data.
+We will demonstrate the relationship between PCA and the SVD.
+
+t-SNE is an alternative nonlinear method for dimensionality reduction.
 ::::
 
-::: {.content-visible when-profile="web"}
-An excellent reference for the following section is at this [link](https://liorpachter.wordpress.com/2014/05/26/what-is-principal-component-analysis/).
+# PCA 
 
-Some figures and discusion are taken from there.
-:::
+## Dimensionality Reduction 
 
-::: {.content-visible when-profile="slides"}
-## Dimensionality reduction continued
-:::
+Input: $\mathbf{x}_1,\ldots, \mathbf{x}_m$ with  $\mathbf{x}_i \in \mathbb{R}^n \: \: \forall \: i \in \{1, \ldots, n\}.$
 
-In order to better understand how the SVD provides us with this optimal transformation, let's consider the following criterion for a $k$-dimensional transformation:
+Output: $\mathbf{y}_1,\dots, \mathbf{y}_m$  with  $\mathbf{y}_i \in \mathbb{R}^d \: \: \forall \: i \in \{1, \dots, n\}$. 
```

Comment:
I'm confused by the indexing and dimensionality. Are the $x_i$ the column vectors of the matrix? Then aren't they in $R^m$?. And same for $y_i$? Maybe the indexing on the matrix equation below is wrong?

Reply:
Added clarifying remarks below.